File: permission/src/main/java/com/cn/permission/aop/listener/PermissionListener.java
package com.cn.permission.aop.listener;

/**
 * @Author: 崔宁 github: https://github.com/Cuinings
 * @Email: 1015597172@qq.com
 * @Date: 2020/6/3 5:59 PM
 * @Description:
 */
public interface PermissionListener {

    void success();//成功

    void cancel();//拒绝

    void denied();//拒绝（不在提示）
}

File: permission_kotlin/src/main/java/com/cn/permission_kotlin/aop/listener/PermissionListener.kt
package com.cn.permission_kotlin.aop.listener

/**
 * @Author:          崔宁 github: https://github.com/Cuinings
 * @Email:          1015597172@qq.com
 * @Date:           2020/6/4 2:29 PM
 * @Description:
 */
interface PermissionListener {

    fun success()

    fun cancel()

    fun denied()
}